### PR TITLE
btrfs: add page

### PR DESCRIPTION
**Pull request by ekickx**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/btrfs.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# btrfs
+
+> A modern copy on write (CoW) filesystem for Linux.
```
Comment (schneiderl) [inline pages/linux/btrfs.md]: We tend to omit adjectives ("modern", "open source", "top quality") in the main description. Stick to describing what it is and what it does.

File: pages/linux/btrfs.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Create subvolume:
+
+`sudo btrfs subvolume create {{path/to/subvolume}}`
+
+- List subvolume:
+
+`sudo btrfs subvolume list {{path/to/moount_point}}`
```
Comment (schneiderl) [inline pages/linux/btrfs.md]: Did you mean...
```suggestion
`sudo btrfs subvolume list {{path/to/mount_point}}`
```
Or moount_point is really something?

Comment (ekickx) [reply pages/linux/btrfs.md]: oh yes, my mistake

File: pages/linux/btrfs.md (hunk 1 above)
Comment (ekickx) [reply pages/linux/btrfs.md]: I've just changed it. Can you review it again? Sorry for my messy commit. Still, haven't used with pr :sweat_smile:

Comment (schneiderl) [reply pages/linux/btrfs.md]: Hi, 
No problem, we can change the commit message once your PR is merged :)

**Comment by sbrl** [comment]
Oops! Spotted a teensy mistake that I ninja-fixed before merging, if  that's ok @ekickx?

**Comment by ekickx** [comment]
> Oops! Spotted a teensy mistake that I ninja-fixed before merging, if that's ok @ekickx?

No problem. Thanks for accepting my pr 😊

**Pull request merged**